### ISSUE TITLE
Tag for release 0.2 is not pushed to github

### ISSUE THREAD

**Issue by ullgren** [mulesoft-labs/raml-java-client-generator#33]
**Describe the bug**
Tag for release 0.2 is not pushed to github

**To Reproduce**
Check the Mulesoft maven release repo at https://repository.mulesoft.org/nexus/content/repositories/releases/org/mule/raml/codegen/raml-client-generator-core/ and note that 0.2 is released.

Look for the tag in the git repo:
```
ponu@bravo:~/raml-java-client-generator (master u=) $ git remote -v
origin	git@github.com:mulesoft-labs/raml-java-client-generator.git (fetch)
origin	git@github.com:mulesoft-labs/raml-java-client-generator.git (push)
ponu@bravo:~/raml-java-client-generator (master u=) $ git fetch --all
Fetching origin
ponu@bravo:~/raml-java-client-generator (master u=) $ git tag -l
v0.1
v0.1.1
```

**Expected behavior**
Since 0.2 is released to maven repository the tag v0.2 should exist and be pushed to github.

**Screenshots**
See above

**Comment by hvm2hvm** [comment]
I'm not sure if this is right place to report this but I have a problem with getting the 0.2 version from maven.

**Comment by ullgren** [comment]
Since the `raml-client-generator-core` is not pushed to maven central you need to have mulesofts maven repo defined in your pom.xml (or your settings.xml).

    <repository>
            <id>mulesoft-releases</id>
            <name>MuleSoft Repository</name>
            <url>https://repository.mulesoft.org/releases/</url>
            <layout>default</layout>
    </repository>

**Comment by hvm2hvm** [comment]
It didn't work. I opened a new issue and deleted my previous comment.

**Comment by machaval** [comment]
I've released the 0.3 version

**Comment by ullgren** [comment]
The problem is that the 0.2 and 0.3 tags has not been pushed to the github repository. 

https://github.com/mulesoft-labs/raml-client-generator/tags

ould you please issue a "git push --tags" to share the tag with the rest of us
